Core/Src/stateMachine.c

```diff
@@ -291,6 +292,29 @@ uint32_t sm_fault_eval(fault_eval_t* index)
 	return 0;
 }
 
+void sm_nc_fault_collect(nc_fault_collection_t *nc_fault_data) {
+
+	bool condition;
+
```

Comment:
This was on me for not mentioning, so my bad, but for these, there shouldnt be any need for our timer system/debounce. Thinking about a fault like a CAN message failure - that would be binary, it either happens or doesnt, and so no timer debounce should be needed.

This is different than our ciritcal faults because those constantly stream data, and if we get .0001s of bad data, we want to start a timer to make sure that data stays bad for a longer period of time. So, i think this part can be scrapped